### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by cruessler**
This is just one sentence yet, but I wanted to quickly open the PR early as a place to collect ideas and feedback.

I was thinking of maybe referencing:

- the “Build” section in `README.md`
- the Discord and the fact that we’re happy to help and guide newcomers
- the fact that there’s a label “good first issue”

Let me know what you think!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@
 12. [Sponsoring](#sponsoring)
 13. [Inspiration](#inspiration)
 14. [Contributors](#contributors)
```
Comment (extrawurst) [inline README.md:41]: lets keep the contributors at the bottom of the page, that seems common practice in other OSS projects

**Comment by cruessler** [comment]
@extrawurst I updated the PR, but I don’t know how to link to a Discord server.

**Comment by extrawurst** [comment]
@cruessler its this one: https://discord.gg/rZv4uxSQx3 (you see it in the intro channel on the top, in the channel description)

**Comment by cruessler** [comment]
@extrawurst Thanks for the link, I’ve updated the PR!

**Comment by extrawurst** [comment]
Thank you @cruessler !

**Pull request merged**